versioned_docs/version-v2.18.x/user-guide/authenticating-with-client-certificates.md

```diff
@@ -52,19 +52,50 @@ Register the client certificate with the user ID in your ESM. The following comm
 
   **Example command in RACF:**  
 
-  `RACDCERT ADD(<dataset>) ID(<userid>) WITHLABEL('<label>') TRUST` 
+  ``` 
+  RACDCERT ADD(<dataset>) ID(<userid>) WITHLABEL('<label>') TRUST
+  SETROPTS RACLIST(DIGTCERT, DIGTRING) REFRESH
+  ```
+  Alternatively, in case you are using the internal API ML mapper, you can use the following command:
+  
+  ```
+  RACDCERT ID(<userid>) MAP 
+  SDNFILTER('<subject's-distinguished-name-filter>')
+  WITHLABEL('<label>')
+  ```
+  **Note:**  The `RACDCERT MAP` command allows mapping a certificate to a user without adding the X.509 certificate to the ESM database. While this approach is more convenient, it could be considered less secure than adding the certificate to the ACID as it offers better control and protection.
 
   **Example command in ACF2:** 
 
   `INSERT <userid>.<certname> DSNAME('<dataset>') LABEL(<label>) TRUST`
 
-  **Example command in Top Secret:** 
```

Comment:
Should we state that using the internal API Mapper is the recommended approach?

Reply:
We state it in some places, but it seems that not everywhere.

Reply:
I will add the same note in this paragraph as well